core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -359,6 +359,11 @@ package object config {
       .timeConf(TimeUnit.MILLISECONDS)
       .createWithDefaultString("10s")
 
+  private[spark] val DRIVER_METRICS_POLLING_INTERVAL =
+    ConfigBuilder("spark.driver.metrics.pollingInterval")
+      .version("4.1.0")
+      .fallbackConf(EXECUTOR_HEARTBEAT_INTERVAL)
```

Comment:
Could you add more fields like `spark.executor.metrics.pollingInterval`, e.g., `.doc` and `.timeConf`? In addition, it would be great if we can place this new configuration to somewhere near the driver configs live instead of here.

https://github.com/apache/spark/blob/86dad8358a42d3c113a6268e9d495e0764b1b85f/core/src/main/scala/org/apache/spark/internal/config/package.scala#L369-L376